src/renderer/kube-object.store.ts

```diff
@@ -31,16 +31,16 @@ import { IKubeApiQueryParams, KubeApi, parseKubeApi } from "./api/kube-api";
 import type { KubeJsonApiData } from "./api/kube-json-api";
 import { Notifications } from "./components/notifications";
 
-export interface KubeObjectStoreLoadingParams {
+export interface KubeObjectStoreLoadingParams<K extends KubeObject> {
   namespaces: string[];
-  api?: KubeApi;
+  api?: KubeApi<K>;
   reqInit?: RequestInit;
 }
 
-export abstract class KubeObjectStore<T extends KubeObject = any> extends ItemStore<T> {
+export abstract class KubeObjectStore<K extends KubeObject> extends ItemStore<K> {
   static defaultContext = observable.box<ClusterContext>(); // TODO: support multiple cluster contexts
 
-  abstract api: KubeApi<T>;
+  abstract api: KubeApi<K>;
```

Comment:
All changes in this file after this line are just because `T` is now `K`?

Reply:
Mostly

Reply:
These kind of changes makes reviewing hard. What's wrong with `T ` and why we wanted to change existing practice (single letter vs word) in this PR 🤔

Reply:
I can revert them, I change it because `K` is closer to `KubeObject` which is what it represents.